README.md

```diff
@@ -63,6 +63,11 @@ Compliance levels can also be updated through the `testing.accessibility.complia
 
 The test file lives in `/tests/accessibility/compliance/pa11y.js` if any edits are needed (such as staging credentials, if you're running tests in an environment that requires authentication).
 
+## Browsersync
```

Comment:
@colorful-tones You can remove this, it's just info about Browsersync - Just a note about the local URL will suffice.